spec/rails_helper.rb

```diff
@@ -45,6 +44,12 @@
   # https://relishapp.com/rspec/rspec-rails/docs
   config.infer_spec_type_from_file_location!
 
+  # Enable aggregate failures except for system tests or explicitly disabled at the spec level.
+  # See http://rspec.info/blog/2015/06/rspec-3-3-has-been-released/#expectations-new-aggregratefailures-api
+  config.define_derived_metadata do |meta|
+    meta[:aggregate_failures] = true unless meta.key?(:aggregate_failures) || meta[:type] == :system
```

Comment:
We need this temporary workaround for screenshots to be output.

@mattbrictson has a PR to address the issue: https://github.com/rspec/rspec-rails/pull/1907

Once the PR is merged we can revert the config change and always aggregate failures (except in the presence of `aggregate_failures` metadata).